pkgs/applications/networking/xpipe/default.nix

```diff
@@ -0,0 +1,71 @@
+{ stdenv, lib, fetchzip, makeDesktopItem }:
+
+let
+  inherit (stdenv.hostPlatform) system;
+  throwSystem = throw "Unsupported system: ${system}";
+
+  arch = {
+    x86_64-linux = "x86_64";
+    aarch64-linux = "arm64";
+  }.${system} or throwSystem;
+
+  sha256 = {
+    x86_64-linux = "VmJa0DqnFSzq4wJ+HchguDK4tMeKP/XaE2MidXfootM=";
+    aarch64-linux = "LdXfhZK4n8HqN1u/rKIIZhFk7fREXbTuAzyvvhFR6Lo=";
+  }.${system} or throwSystem;
+
+  displayname = "XPipe";
+
+in stdenv.mkDerivation rec {
```

Comment:
I think `stdenv` should be changed to `stdenvNoCC` if you're not planning to build the executable here.